tasks_management/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+
+from django.contrib.auth.models import AnonymousUser
+from django.db.models import Q
+
+from core.schema import OrderedDjangoFilterConnectionField
+from core.utils import append_validity_filter
+from tasks_management.apps import TasksManagementConfig
+from tasks_management.gql_mutations import CreateTaskGroupMutation, UpdateTaskGroupMutation, DeleteTaskGroupMutation
+from tasks_management.gql_queries import TaskGroupGQLType, TaskExecutorGQLType
+from tasks_management.models import TaskGroup, TaskExecutor
+
+import graphene_django_optimizer as gql_optimizer
+
+
+class Query(graphene.ObjectType):
+    module_name = "tasks_management"
+
+    task_group = OrderedDjangoFilterConnectionField(
+        TaskGroupGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        dateValidFrom__Gte=graphene.DateTime(),
+        dateValidTo__Lte=graphene.DateTime(),
+        client_mutation_id=graphene.String(),
+    )
+    task_executor = OrderedDjangoFilterConnectionField(
+        TaskExecutorGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        dateValidFrom__Gte=graphene.DateTime(),
+        dateValidTo__Lte=graphene.DateTime(),
+        client_mutation_id=graphene.String(),
+        taskGroupIdStr=graphene.String(),
```

Comment:
Is this 'taskGroupIdStr' the same as 'taskGroupIdString'? Please use full expression for naming this argument if it is possible.

Reply:
resolved